website/versioned_docs/version-latest/reference/templating.mdx

```diff
@@ -115,7 +115,7 @@ special variable will be overridden.
 | `TIMESTAMP`        | The date object of the greatest timestamp of the files listed in `sources`. Only available within the `status` prop and if method is set to `timestamp`. |
 | `TASK_VERSION`     | The current version of task.                                                                                                                             |
 | `ITEM`             | The value of the current iteration when using the `for` property. Can be changed to a different variable name using `as:`.                               |
-
+| `EXIT_CODE`	     | The exit code of the last failed task. Can be used inside a defer statement to perform different operations according to the exit code.                 	|
```

Comment:
This should be done only in `docs/reference/templating.mdx` and reverted from this file
versioned_docs is generated and will be overriden